### PR TITLE
FOUNDATIONS: Add Intern

### LINKED ISSUE #48: FOUNDATIONS: Add Intern

### PR DESCRIPTION
**Pull request by Pwongchaiya**
closed #48

### CODE REVIEW

File: InternTrack.Core.Api.Tests.Unit/Services/Foundations/Interns/InternServiceTests.Exceptions.Add.cs [hunk 1]
```diff
@@ -31,9 +31,9 @@ public async Task ShouldThrowCriticalDepdnencyExceptionOnAddIfSqlErrorOccursAndL
             var expectedInternDependencyExcetpion =
                 new InternDependencyException(failedInternStorageException);
 
-            this.storageBrokerMock.Setup(broker =>
-                broker.InsertInternAsync(It.IsAny<Intern>()))
-                    .ThrowsAsync(sqlException);
+            this.dateTimeBrokerMock.Setup(broker =>
```
Comment (glhays) [inline InternTrack.Core.Api.Tests.Unit/Services/Foundations/Interns/InternServiceTests.Exceptions.Add.cs:34]: Fix margins

**Comment by glhays** [comment]
FYI, Don't forget to close the issue

**Comment by Pwongchaiya** [comment]
@glhays I closed this one already. Had to redo it because did it incorrectly the first time.

**Comment by glhays** [comment]
@Pwongchaiya 
Isn't this PR for Issue 48? Your not closing it if this PR is merged ^^^^

**Comment by Pwongchaiya** [comment]
@glhays Its a brand new PR from the same branch that merged. The issue has already been closed

**Review by glhays — APPROVED** [review]
@hassanhabib My review is done on this PR, LGTM

**Comment by Pwongchaiya** [comment]
Screenshot (136)

**Pull request merged**